src/responder/nss/nsssrv_mmap_cache.c

```diff
@@ -445,6 +455,14 @@ static errno_t sss_mc_find_free_slots(struct sss_mc_ctx *mcc,
     } else {
         cur = mcc->next_slot;
     }
+    if (cur == 0) {
+        /* inform only once per full loop to avoid excessive spam */
+        DEBUG(SSSDBG_IMPORTANT_INFO, "mmap cache of type '%s' is full",
+              mc_type_to_str(mcc->type));
```

Comment:
missing \n st the end of the DEBUG message, I think it is not needed for sss_log.

Reply:
Thanks, fixed.